src/cgutils.cpp

```diff
@@ -1424,37 +1415,37 @@ static jl_cgval_t emit_getfield_knownidx(const jl_cgval_t &strct, unsigned idx,
         bool isboxed;
         Type *lt = julia_type_to_llvm((jl_value_t*)jt, &isboxed);
         if (isboxed) {
-            Value *ptr = decay_derived(data_pointer(strct, ctx, T_pint8));
+            Value *ptr = decay_derived(data_pointer(ctx,strct, T_pint8));
```

Comment:
I blame `sed` :P

I attempted to clean everything up as I went, but let me know if I missed any others. Although, really would probably more effective for your time to review #15591 and push that forwards towards merging.